### DISCUSSION TITLE
Change TastytradeAuth to receive base url as an argument to allow for cert environment

### DISCUSSION THREAD

**Discussion by cskolaut** [peter-oroszvari/tastytrade-api#5] [Ideas]
When testing/developing against the TastyTrade API, you should be able to use the https://api.cert.tastyworks.com URL. 
Allow for authenticating against the cert environment

**Comment by peter-oroszvari** [comment]
Thanks for the suggestion, will implement soon.

**Comment by peter-oroszvari** [comment]
I've just updated the __init.py__ in the root folder, added a variable, and updated the URLs of authentication.py
